BUILD

```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Please don't set the default visibility to public. It makes it far too easy to leak targets. I say this as someone who has been surprised multiple times to learn a target was public and then needed to clean it up. Yes, I know it'll be annoying now because everything is actually public, but once we start adding tests and using java_libraries it is asking for trouble.